packages/dcos-integration-test/extra/test_service_discovery.py

```diff
@@ -170,20 +172,20 @@ def get_dns_addresses_by_app_name(app_name):
     return DNSAddresses(container_ips, agent_ips, auto_ips)
 
 
-def get_marathon_addresses_by_service_points(service_points):
+def get_marathon_addresses_by_service_points(service_points: list) -> MarathonAddresses:
     marathon_host_addrs = frozenset([point.host for point in service_points])
     marathon_ip_addrs = frozenset([point.ip for point in service_points])
     return MarathonAddresses(marathon_host_addrs, marathon_ip_addrs)
 
 
-def get_dcos_dns_records():
+def get_dcos_dns_records() -> Any:
```

Comment:
Isn't this returning a `Dict`?

Reply:
You are right but also None as an option, in this case `Optional[dict]` will be solution.
Also I can not find return definition of response.json() method but I assune that it can be not only dict so I will add type hint as result of this function like bellow:
```
# type: Optional[dict]
```